examples/sandbox/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hdwallet-sandbox",
-  "version": "1.3.0",
```

Comment:
can this be a minor version bump? breaking changes?

Reply:
or patch rather

Reply:
k